### PR TITLE
Add new option `follow-symlinks` to traverse symbolic links

### PR DESCRIPTION
**Pull request by rashil2000**
As per the discussion in #300

### CODE REVIEW

File: doc/telescope-file-browser.txt
Comment (jamestrew) [inline doc/telescope-file-browser.txt:1]: the help docs for telescope are all auto-generated from docstrings so this is not necessary. should be fine though i think.

File: README.md [hunk 1]
```diff
@@ -106,6 +106,7 @@ require("telescope").setup {
       select_buffer = false,
       hidden = { file_browser = false, folder_browser = false },
       -- respect_gitignore
+      -- follow_symlinks
```
Comment (jamestrew) [inline README.md:109]: this can just be `follow_symlinks = false` uncommented

**Review by jamestrew — COMMENTED** [review]
Thanks I think we're looking mostly good here. Just a couple of small tweaks and I'll merge this.

**Comment by rashil2000** [comment]
Done, thanks for the review :)

**Review by jamestrew — APPROVED** [review]

**Pull request merged**